openadapt/app/dashboard/api/index.py

```diff
@@ -0,0 +1,8 @@
+from fastapi import FastAPI
+
+app = FastAPI()
+
+
+@app.get("/api/python")
+def hello_world():
+    return {"message": "Hello World"}
```

Comment:
I think to get this merged, there should be some minimal functionality, e.g:

- list recordings (e.g. https://github.com/OpenAdaptAI/OpenAdapt/blob/main/openadapt/app/tray.py#L157)
- start recording (e.g. https://github.com/OpenAdaptAI/OpenAdapt/blob/main/openadapt/app/cards.py#L79)
- stop recording (e.g. https://github.com/OpenAdaptAI/OpenAdapt/blob/main/openadapt/app/cards.py#L68)